legal/terms/terms_of_sale.rst

```diff
@@ -0,0 +1,50 @@
+.. _terms_of_sale:
+
+=====================
+General Terms of Sale
+=====================
+
+The client explicitly waives its own standard terms and conditions, even if
+these were drawn up after these standard terms and conditions of sale. In
+order to be valid, any derogation must be expressly agreed to in advance in
+writing.
+
+Our invoices are payable within 21 working days, unless another payment
+timeframe is indicated on either the invoice or the order. In the event of
+non-payment by the due date, Odoo S.A. reserves the right to request a fixed
+interest payment amounting to 15% of the sum remaining due. Odoo S.A. will be
+authorised to suspend any provision of services without prior warning in the
+event of late payment.
+
+If a payment is still outstanding more than sixty (60) days after the due
+payment date, Odoo S.A. reserves the right to call on the services of a debt
+recovery company. All legal expenses will be payable by the client.
+
+Certain countries apply withholding at source on the amount of invoices, in
+accordance with their internal legislation. Any withholding at source will be
+paid by the client to the tax authorities. Under no circumstances can Odoo S.A.
+become involved in costs related to a country's legislation. The amount of the
+invoice will therefore be due to Odoo S.A. in its entirety and does not include
+any costs relating to the legislation of the country in which the client is
+located.
+
+Odoo S.A. undertakes to do its best to supply services in due time in accordance
+with the agreed timeframes. However, none of its obligations can be considered
+as being an obligation to achieve results. Odoo S.A. cannot, under any
+circumstances, be required by the client to appear as a third party in the
+context of any claim for damages filed against the client by an end consumer.
+
+In order for it to be admissible, Odoo S.A. must be notified of any claim by
+means of a letter sent by recorded delivery to its registered office within 8
+days of the delivery of the goods or the provision of the services.
+
+To the maximum extent permitted by law, the aggregate liability of each party
+together with its affiliates arising will not exceed 50% of the total amount
+paid by the Customer under this Agreement during the 12 months immediately
+preceding the date of the event giving rise to such claim. Multiple claims shall
+not enlarge this limitation.
```

Comment:
Links to FR/EN terms of amazon:
- https://www.amazon.co.uk/gp/help/customer/display.html?nodeId=201909000
- https://www.amazon.fr/gp/help/customer/display.html?nodeId=548524